skiko/src/macosX64Main/cpp/generated/Canvas.cc

```diff
@@ -80,29 +81,19 @@ extern "C" void org_jetbrains_skia_Canvas__1nDrawOval
     canvas->drawOval({left, top, right, bottom}, *paint);
 }
 
-
-extern "C" void org_jetbrains_skia_Canvas__1nDrawRRect
-  (kref __Kinstance, jlong canvasPtr, jfloat left, jfloat top, jfloat right, jfloat bottom, jfloatArray jradii, jlong paintPtr) {
-    throw std::runtime_error("TODO: implement org_jetbrains_skia_Canvas__1nDrawRRect");
-}
-     
-#if 0 
 extern "C" void org_jetbrains_skia_Canvas__1nDrawRRect
-  (kref __Kinstance, jlong canvasPtr, jfloat left, jfloat top, jfloat right, jfloat bottom, jfloatArray jradii, jlong paintPtr) {
+  (kref __Kinstance, jlong canvasPtr, jfloat left, jfloat top, jfloat right, jfloat bottom, jfloatArray jradii, jint size, jlong paintPtr) {
     SkCanvas* canvas = reinterpret_cast<SkCanvas*>(static_cast<uintptr_t>(canvasPtr));
     SkPaint* paint = reinterpret_cast<SkPaint*>(static_cast<uintptr_t>(paintPtr));
-    canvas->drawRRect(skija::RRect::toSkRRect(env, left, top, right, bottom, jradii), *paint);
+    canvas->drawRRect(skija::RRect::toSkRRect(left, top, right, bottom, jradii, size), *paint);
 }
-#endif
-
-
 
 extern "C" void org_jetbrains_skia_Canvas__1nDrawDRRect
   (kref __Kinstance, jlong canvasPtr,
    jfloat ol, jfloat ot, jfloat oright, jfloat ob, jfloatArray ojradii,
    jfloat il, jfloat it, jfloat ir, jfloat ib, jfloatArray ijradii,
    jlong paintPtr) {
-    throw std::runtime_error("TODO: implement org_jetbrains_skia_Canvas__1nDrawDRRect");
+    TODO("implement org_jetbrains_skia_Canvas__1nDrawDRRect");
```

Comment:
Do we skip those just to avoid commonization?

Reply:
Ah, I didn't realize there was only of left using skija::RRect::toSkiaRRect. Converted this one too.